### PR TITLE
EN-4270 - update round info in elastic search for metachain

### PR DESCRIPTION
**Pull request by miiu96**
Add round info in elastic search when a meta block was committed successfully.

### CODE REVIEW

File: process/block/metablock.go [hunk 1]
```diff
@@ -261,18 +261,27 @@ func (mp *metaProcessor) checkAndRequestIfShardHeadersMissing(round uint64) {
...
 	}
 
-	//TODO: maybe index metablocks also?
+	publicKeys, err := mp.nodesCoordinator.GetValidatorsPublicKeys(metaBlock.GetPrevRandSeed(), metaBlock.GetRound(), sharding.MetachainShardId)
+	if err != nil {
+		return
+	}
+	signersIndexes := mp.nodesCoordinator.GetValidatorsIndexes(publicKeys)
+
+	saveRoundInfoInElastic(mp.core.Indexer(), mp.nodesCoordinator, sharding.MetachainShardId, metaBlock, lastMetaBlock, signersIndexes)
```
Comment (bogdan-rosianu) [inline process/block/metablock.go]: add empty line before `signerIndexes` and remove empty line before `saveRoundInfo..`

Comment (miiu96) [reply process/block/metablock.go]: done.

Comment (bogdan-rosianu) [reply process/block/metablock.go]: not done yet

Comment (miiu96) [reply process/block/metablock.go]: done

**Pull request merged**